dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,202 @@
+CWMCM0013E.jsonrpc.internal.error.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# for server
+# {0} = tool name
+# {1} = exception details
+CWMCM0014E.internal.server.error.detailed=CWMCM0014E: The {0} tool method threw an unexpected exception. The exception was {1}
+CWMCM0014E.internal.server.error.detailed.explanation=The tool method threw an unexpected exception.
+CWMCM0014E.internal.server.error.detailed.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# for client
+CWMCM0015E.internal.server.error=CWMCM0015E: An internal server error occurred
+CWMCM0015E.internal.server.error.explanation=An internal server error occurred
+CWMCM0015E.internal.server.error.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# for server
+# do not translate bean
+# {0} = bean error details
+# {1} = tool name
+CWMCM0016W.bean.release.fail=CWMCM0016W: An error occurred when releasing the CDI bean after calling the {1} tool. The error is: {0}
+CWMCM0016W.bean.release.fail.explanation=When calling a tool method, an instance of the CDI bean which declares the method is acquired before calling the method and released afterwards. An error occurred when releasing the bean instance.
+CWMCM0016W.bean.release.fail.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# for server and client
+# do not translate MCP-Protocol-Version, MCP
+# {0} = List of supported MCP-Protocol-Version values
+CWMCM0017E.unsupported.mcp.http.version=CWMCM0017E: An unsupported MCP-Protocol-Version header was provided. Supported values: {0}
+CWMCM0017E.unsupported.mcp.http.version.explanation=A unsupported MCP-Protocol-Version header was provided in the request.
+CWMCM0017E.unsupported.mcp.http.version.useraction=Provide an MCP version in the header that's in the list of supported values.
+
+#for client
+#do not translate Method, RequestURI, RequestQuery
+# {0} = Method
+# {1} = RequestURI
+# {2} = RequestQuery
+CWMCM0018E.unexpected.server.error=CWMCM0018E: Unexpected Server Error. Method={0} RequestURI={1} RequestQuery={2}
+CWMCM0018E.unexpected.server.error.explanation=An unexpected  server error occurred whilst processing the request. The request could not be completed.
+CWMCM0018E.unexpected.server.error.useraction=No action is required.
+
+# for server and client
+#do not translate Method, RequestURI, RequestQuery
+# {0} = Method
+# {1} = RequestURI
+# {2} = RequestQuery
+# {2} = ExceptionMessage
+CWMCM0019E.unexpected.server.error.exception=CWMCM0019E: Unexpected Server Error. Method={0} RequestURI={1} RequestQuery={2}, Exception: {3}
+CWMCM0019E.unexpected.server.error.exception.explanation=An unexpected  server error occurred whilst processing the request. The request could not be completed.
+CWMCM0019E.unexpected.server.error.exception.useraction=No action is required.
```

Comment:
This one's now just for the server, right?

Double space between "unexpected" and "server" in the explanation.

`# {2} = ExceptionMessage` should be `# {3} = ExceptionMessage`